src/compiler/transformers/es2015.ts

```diff
@@ -4074,9 +4074,13 @@ namespace ts {
         priority: 0,
         text: `
             var __extends = (this && this.__extends) || (function () {
-                var extendStatics = Object.setPrototypeOf ||
-                    ({ __proto__: [] } instanceof Array && function (d, b) { d.__proto__ = b; }) ||
-                    function (d, b) { for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p]; };
+                function extendStatics(d, b) {

```

Comment:
Could you make this a `var` per https://github.com/Microsoft/TypeScript/pull/24244#issuecomment-390468895?